src/pages/invoices/index/Invoices.tsx

```diff
@@ -8,21 +8,64 @@
  * @license https://www.elastic.co/licensing/elastic-license
  */
 
+import invoiceStatus from 'common/constants/invoice-status';
+import { date } from 'common/helpers';
+import { useCurrentCompanyDateFormats } from 'common/hooks/useCurrentCompanyDateFormats';
 import { useTitle } from 'common/hooks/useTitle';
+import { DataTable, DataTableColumns } from 'components/DataTable';
 import { Default } from 'components/layouts/Default';
+import { StatusBadge } from 'components/StatusBadge';
 import { useTranslation } from 'react-i18next';
-import { Link as RouterLink } from 'react-router-dom';
+import { generatePath, Link } from 'react-router-dom';
 
 export function Invoices() {
   const [t] = useTranslation();
-
+  const { dateFormat } = useCurrentCompanyDateFormats();
   const pages = [{ name: t('invoices'), href: '/invoices' }];
-
+  const columns: DataTableColumns = [
+    {
+      id: 'status_id',
+      label: t('status'),
+      format: (value) => <StatusBadge for={invoiceStatus} code={value} />,
+    },
+    {
+      id: 'number',
+      label: t('number'),
+    },
+    {
+      id: 'client_id',
+      label: t('client'),
+      format: (value, resource) => (
+        <Link to={generatePath('/clients/:id', { id: resource.client.id })}>
```

Comment:
Wrong component. We should use <Link> from 'components/forms'. At the moment there's no visual identification it's clickable as it appears as any other text in the row.